docs/notebooks/multi_trust_region.pct.py

```diff
@@ -0,0 +1,164 @@
+# ---
```

Comment:
(haven't looked at yet, but you'll need to update tutorials.rst so there's a link to it; probably worth manually running the quality checks job on it too, or building locally, to catch any sphinx errors)

Reply:
I have added a link to that file. I had already run quality checks, but now locally also ran sphinx on it.